modules/rhpam-apb/roles/deploy-businesscentral-monitoring/templates/businesscentral-monitoring-dc.yml.j2

```diff
@@ -200,7 +200,7 @@ spec:
               mountPath: "/etc/businesscentral-certs"
               readOnly: true
             - name: businesscentral-data
```

Comment:
This one needs to be renamed as well? @errantepiphany

Reply:
would make sense move it to /opt/kie/certificates/{businesscentral,kieserver}

Reply:
Just checking in... I like /opt/kie/data/certs/* . Keeping everything in /opt/kie will be very helpful and organized.

Reply:
@spolti I was talking about the `businesscentral-data`, but you made a good point. The problem is, this mountPath should be changed in every template as well. Don't you think it's better to open a different JIRA (or PRs) to make this change?

Reply:
@errantepiphany ^

Reply:
@ricardozanini I am fine with the same jira, as long as it gets done across each of the images.